homeassistant/components/backup/manager.py

```diff
@@ -114,6 +114,12 @@ class BackupManagerState(StrEnum):
     RESTORE_BACKUP = "restore_backup"
 
 
+class BackupPlatformState(StrEnum):
+    """Backup platform type."""
+
+    LOADED = "loaded"
```

Comment:
If there's only one state, why do we need this state on the event?

Reply:
you're right :+1: